fakebook/.vscode/settings.json

```diff
@@ -0,0 +1 @@
+123,125
```

Comment:
?

Reply:
Somehow a file that should have been git ignored got into this PR. Removed